test/parallel/test-fs-read-position-validation.mjs

```diff
@@ -45,57 +33,35 @@ async function testValid(position, allowedErrors = []) {
         fs.read(fd, { buffer, offset, length, position }, common.mustSucceed());
       }
     } finally {
-      fs.close(fd, common.mustSucceed());
+      fs.close(fd, common.mustSucceed(callback));
```

Comment:
Shouldn't we wait for the read operations to finish before closing the file?

Reply:
Right, this is exactly what caused the problem initially.